### PR TITLE
Dispatch actions

### PR DESCRIPTION
**Pull request by DuaneDave**
### In this PR, actions were dispatched to enable:

- The add functionality
- The delete functionality
- The check status functionality

### CODE REVIEW

File: src/components/books/Category.jsx [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+
+  const checkStatusHandler = () => {
+    dispatch({
+      type: 'bookstore/categories/CHECK_STATUS',
+    });
+  };
+
+  return (
+    <Button type="button" title="CHECK STATUS" onAction={checkStatusHandler} />
+  );
```
Comment (cynthiainga) [inline src/components/books/Category.jsx]: - [ ] When testing your app I noticed that the `CHECK STATUS` button does not work, when I click on the button nothing happens. The functionality of this button is when the user clicks on it it's supposed to return the `Under construction ` text, please kindly fix that 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @DuaneDave, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] All linters are passing ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/Category.jsx (hunk 1 above)
Comment (DuaneDave) [reply src/components/books/Category.jsx]: I thought of that, but there is no component to display that

File: src/components/books/Category.jsx [hunk 2]
```diff
@@ -11,7 +12,14 @@ function Category() {
...
   return (
-    <Button type="button" title="CHECK STATUS" onAction={checkStatusHandler} />
+    <div>
+      <p>{category}</p>
+      <Button
+        type="button"
+        title="CHECK STATUS"
+        onAction={checkStatusHandler}
+      />
+    </div>
```
Comment (mateo951) [inline src/components/books/Category.jsx:22]: Addressed 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @DuaneDave, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**